assets/js/event-filtering.js

```diff
@@ -0,0 +1,67 @@
+$(function() {
+    var eventsElements = $(".event-tags span");
+    var eventFilterParent = $("#eventFilter");
+    var tags = [];
+
+    for (var i = 0; i < eventsElements.length; i++) {
+        var elem = eventsElements[i];
+        var text = $(elem).text();
+        if (tags.indexOf(text) === -1) {
+            tags.push(text);
+            eventFilterParent.append(
+                '<input type="checkbox" class="mx-2" value="' + text + '">' + text + '</p>'
+            );
+        }
+    }
+
+    $("input[type='checkbox']").on("click", function() {
+        var inputs = $("input[type='checkbox']");
+        var chosenInputs = [];
+
+        for (var z = 0; z < inputs.length; z++) {
```

Comment:
Nit: Any reason why you didn't want to use `var i` like in the `for`-loop above?

Reply:
No reason in particular. Probably a side-effect from something I was doing earlier that had two for loops in the same function. Good catch.

Side note: `z`'s are much cooler than `i`'s 😎